sklearn/metrics/classification.py

```diff
@@ -292,6 +293,204 @@ def confusion_matrix(y_true, y_pred, labels=None, sample_weight=None):
     return CM
 
 
+def multilabel_confusion_matrix(y_true, y_pred, sample_weight=None,
+                                labels=None, samplewise=False):
+    """Compute multi-label confusion matrix label-wisely (default) or
```

Comment:
This summary should be one line. Perhaps just "Compute a confusion matrix for each class or sample